src/sentry/static/sentry/app/views/stream/actions.jsx

```diff
@@ -277,6 +277,12 @@ const StreamActions = createReactClass({
     });
   },
 
+  onCreateIncident() {
+    const {organization} = this.props;
+    const issues = this.state.selectedIds;
+    openCreateIncidentModal({organization, issues: Array.from(issues)});
```

Comment:
What's the purpose of calling `Array.from`?

Reply:
It's a `Set`